src/olympia/files/models.py

```diff
@@ -112,7 +113,7 @@ class File(OnChangeMixin, ModelBase):
     status = models.PositiveSmallIntegerField(
         choices=STATUS_CHOICES.items(), default=amo.STATUS_AWAITING_REVIEW
     )
-    datestatuschanged = models.DateTimeField(null=True, auto_now_add=True)
+    datestatuschanged = models.DateTimeField(null=True, default=timezone.now)
```

Comment:
Not saying this is a bad change to make ... but we could just special case `datestatuschanged` field in `file_factory` - pop the value and set it after the create, and so avoid any database/model change to prod.